src/backups.py

```diff
@@ -317,3 +338,236 @@ def _post_backup(self) -> Tuple[bool, str]:
             return False, "Error setting instance option tag:_hidden"
 
         return True, None
+
+    # ------------------ Perform Restore ------------------
+
+    def _pre_restore_checks(self, event: ActionEvent) -> bool:
```

Comment:
looks like we have two different functions which do pre restore checks `_can_unit_restore_backup` and `_pre_restore_checks`.
any reason not to move `_can_unit_restore_backup` code into `_pre_restore_checks`?

Reply:
addressed in 86b1309ee0fe242d44ce2a0d3360a0974b520bfd (no reason not to move it)